style.css

```diff
@@ -0,0 +1,1229 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+.logo {
+  display: none;
+}
+
+.menu-bigscreen {
+  display: none;
+}
+
+.up-nav {
+  display: none;
+}
+
+.menu {
+  display: none;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.menu-mobile {
+  position: relative;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 90%;
+  background-color: #3b3b3f;
+  z-index: 1;
+  opacity: 0;
+  visibility: hidden;
+  transition: all 0.3s ease-in-out;
+  display: none;
+}
+
+.menu-mobile.active {
+  opacity: 0.9;
+  visibility: visible;
+}
+
+.nav-links-mobile {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  gap: 20px;
+  height: 50vh;
+  background-color: #3b3b3f;
+  padding: 15px 15px;
+  list-style: none;
+}
+
+.a-links {
+  display: block;
+  font-size: 30px;
+  text-align: center;
+  color: #f55800;
+  text-decoration: none;
+}
+
+/* Burger and close icon styles */
+.burger {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  cursor: pointer;
+  padding-left: 10px;
+  padding-top: 10px;
+}
+
+.close-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  color: #0d0c0c;
+}
+
+#home {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  position: relative;
+  background-image: url(assets/Frame-25-min-1.jpg.webp);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+#home::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: url(assets/weathered-concrete-surface-wallpaper-backdrop.jpg);
+  background-size: cover;
+  background-position: center;
+  filter: blur(7px);
+  -webkit-filter: blur(7px);
+  opacity: 0.8;
+}
+
+#home > * {
+  position: relative;
+  z-index: 1;
+}
+
+.container {
+  margin: 15px;
+  margin-top: 40px;
+}
+
+.p-1 {
+  font-family: 'Lato', sans-serif;
+  font-size: 25px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.9px;
+  color: #f55800;
+  text-align: center;
+  margin-bottom: 10px;
+  display: inline-block;
+}
+
+.head-1 {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 50px;
+  line-height: 64px;
+  letter-spacing: 0.27px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+  background: linear-gradient(rgba(0, 0.2, 0, 0.2), rgba(0.3, 0, 0, 0.6)), url('assets/SL_060521_43530_09.jpg') repeat;
+  -webkit-background-clip: text;
+  color: transparent;
+  filter: brightness(120%);
+  background-position: center;
+  background-size: cover;
+}
+
+.box-1 {
+  font-family: 'Lato', sans-serif;
```

Comment:
- [ ] It is noticed that `font-family: Lato` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.basic-font` and assigning it wherever possible as a class name in your `html` markup code.
```
.basic-font {
  font-family: Lato;
}

```
- And kindly attend to any repetitions in your code 🙂.